### PR TITLE
(#1917516) Don't emit PropertiesChanged signal needlessly

### PR DESCRIPTION
**Pull request by msekletar**

### CODE REVIEW

**Review by dtardon — APPROVED** [review]
LGTM

**Comment by jamacku** [comment]
@mrc0mmand , could you please review CI. Thank you.

File: src/core/unit.c [hunk 1]
```diff
@@ -270,6 +270,9 @@ int unit_set_description(Unit *u, const char *description) {
         if (isempty(description))
                 s = NULL;
         else {
+                if (streq_ptr(u->description, description)
```
Comment (mrc0mmand) [inline src/core/unit.c:273]: Missing ')'

**Review by mrc0mmand — CHANGES_REQUESTED** [review]

File: src/core/unit.c (hunk 1 above)
Comment (msekletar) [reply src/core/unit.c]: Not quite sure how this happened since I was cherry-picking and this patch applied cleanly IIRC. Fixed now.

**Comment by mrc0mmand** [comment]
The LGTM fail is expected, since the 7.6 branch is missing 6154d33de3f15bbd5d5df718103af9c37ba0a768.

**Pull request merged**